From 96fcd624d9ed05de29d6a3aa6b373a2721d306ec Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Tue, 25 Jun 2024 15:37:11 -0700
Subject: [PATCH] Docs/mm/damon/design: add a sysfs usage link for setting
 DAMON operations set

Readers of the design document's operations set layer section would
wonder how they can set the operations set.  Add a link to a section of
DAMON sysfs interface usage document that provides the answer on the
design doc for easier browsing.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/design.rst | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index e5591f909ebf..4ed08cba314f 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -74,6 +74,10 @@ describe how those work.
  - fvaddr: Monitor fixed virtual address ranges
  - paddr: Monitor the physical address space of the system
 
+To know how user-space can do the configuration via :ref:`DAMON sysfs interface
+<sysfs_interface>`, refer to :ref:`operations <sysfs_context>` file part of the
+documentation.
+
 
  .. _damon_design_vaddr_target_regions_construction:
 
-- 
2.39.2

